x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.js

```diff
@@ -53,7 +57,12 @@ export class BulkUploader {
     });
 
     this._callClusterWithInternalUser = callClusterFactory(server).getCallClusterInternal();
-    this._getKibanaInfoForStats = () => getKibanaInfoForStats(server, kbnServer);
+    this._getKibanaInfoForStats = () => getKibanaInfoForStats(server, kbnServer, {
+      statuses: [{
+        name: 'includes-usage-data',
```

Comment:
This should really only be needed in the backported version. I'm fine leaving it here for now, but we should have a followup commit in master that removes this since we don't need it going forward.